react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -170,14 +182,27 @@ public void editorDidMount(boolean hasUnsupportedBlocks) {
                 }
             }
         });
+
+        ImagePipelineConfig imagePipelineConfig = getImagePipelineConfig(sOkHttpClient);
+        MainReactPackage mainReactPackage = new MainReactPackage(getMainPackageConfig(imagePipelineConfig));
+
         return Arrays.asList(
-                new MainReactPackage(),
+                mainReactPackage,
```

Comment:
We can probably just remove the local variables `imagePipelineConfig` and `mainReactPackage` and just replace this reference in this line here with ```new MainReactPackage(getMainPackageConfig(getImagePipelineConfig(sOkHttpClient)))```

Reply:
Yup, we can do that :)